src/Sdk/DTPipelines/workflow-v1.0.json

```diff
@@ -515,6 +515,20 @@
       "string": {}
     },
 
+    "string-runner-context-no-secrets": {
```

Comment:
i like this.... wondering if there is a specific reason we didn't take this approach with outputs. @TingluoHuang thoughts?

Reply:
discussed offline, we seem to just ignore/warn on client side